pypiserver/templates/deployment.yaml

```
@@ -21,8 +21,15 @@ spec:
       labels:
         app.kubernetes.io/name: {{ template "pypiserver.name" . }}
         app.kubernetes.io/instance: {{ .Release.Name }}
-        {{- if .Values.deployment.labels }}
-        {{- toYaml .Values.deployment.labels | nindent 8 }}
+        {{- if .Values.podLabels }}
+        {{- toYaml .Values.podLabels | nindent 8 }}
+        {{- end }}
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
```

`secret` makes more sense to me (so if we introduce a configmap at some point we could still use checksum/config for that file.)
```suggestion
        checksum/secret: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
```